sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -251,6 +251,12 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val ORC_VECTORIZED_READER_ENABLED =
+    SQLConfigBuilder("spark.sql.orc.enableVectorizedReader")
+      .doc("Enables vectorized orc reader.")
+      .booleanConf
+      .createWithDefault(true)
```

Comment:
I turn it on in order to test if all tests can pass. I will turn it off by default later.